src/Card/MasterCard.java

```diff
@@ -0,0 +1,8 @@
+package Card;
+
+public class MasterCard extends Card{
+    public MasterCard(int cardId, String IBAN, String name)
```

Comment:
[Fix] aceste doua clase, MasterCard si Visa sunt identice. Ai fi putut avea doar un String sau un enum in Card care sa fie type="MASTERCARD" sau "VISA". Daca aceste doua clase nu aduc nimic in plus in materie de campuri, nu isi au rostul ca si clase separate, daca diferenta dintre ele se poate face printr-un simplu nume